### DISCUSSION TITLE
Hx-Boost on Hx-Post?

### DISCUSSION THREAD

**Discussion by jefffdelima** [bigskysoftware/htmx#2252] [General]
Hi Guys,

I am new to HTMX and just wondering if hx-boost works with hx-post even when javascript is off on client's side?

I am trying to imitate the example of Cascading Select, but want to implement fallback, which also works the same thing when using HTMX.

By the way, I am using PHP procedural method as backend.

Thanks, Guy! :)

HTMX rocks!

**Comment by jefffdelima** [comment]
Sorry guys! My question is just so silly... I just realized that everything will be handled by PHP on the backend when JavaScript is off on client's side. I am closing this discussion then.

Again, big apologies, Guys! I am just new to programming.

HTMX Rocks! :)

**Reply by Telroshan** [reply]
Late to the party ; there's no silly question!
Glad you got it figured it out, and that you're enjoying htmx :)
